### PR TITLE
PMM-12194 Create Advisor details page

### PR DESCRIPTION
**Pull request by rnovikovP**
We have received numerous inquiries regarding the details of Advisors and checks inside. Although this information should be automated, we have temporarily documented the list and will update it as new advisors are added.

We propose a separate page in the PMM document to present the list of Advisors and checks according to different technologies. Separating the list by DB technology is crucial for clients as they have full access to Advisors.

### CODE REVIEW

File: docs/get-started/advisors-details.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
+
+## List of Advisors
+
+The [**Advisors**](https://portal.percona.com/advisors) page in the Percona Portal lists the Percona Advisors you can run from PMM. That Page will show the Advisors available for your subscription tier.
+
+For simplicity, we also listening all Advisors and checks on this page:
```
Comment (ademidoff) [inline docs/get-started/advisors-details.md:6]:
```suggestion
For simplicity, we also listing all Advisors and checks on this page:
```

File: docs/get-started/advisors-details.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+| **Connection Security** | Helps identify security issues on network connections and provides recommendations for enhancing security. | Customers only |MySQL, MongoDB|
+
+### Query Advisors
+| Advisor Name | Description | Subscription | Database Technology|
+| :--------- | :---------- | :--- |:--- |
+| **Index Query** | Provides query and index optimization strategies for peak database performance. | Customers only | MySQL, MongoDB, PostgreSQL |
+| **Schema Design Query** | Helps create efficient database schemas by analyzing queries and offering suggestions for optimization. | All Users |MySQL|
+
+## List of checks
+Every Advisor consists of the advisor checks inside.
```
Comment (ademidoff) [inline docs/get-started/advisors-details.md:46]:
```suggestion
Every Advisor consists of one or more Advisor checks. 
```

File: docs/get-started/advisors-details.md [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+
+### Query Advisors
+| Advisor Name | Description | Subscription | Database Technology|
+| :--------- | :---------- | :--- |:--- |
+| **Index Query** | Provides query and index optimization strategies for peak database performance. | Customers only | MySQL, MongoDB, PostgreSQL |
+| **Schema Design Query** | Helps create efficient database schemas by analyzing queries and offering suggestions for optimization. | All Users |MySQL|
+
+## List of checks
+Every Advisor consists of the advisor checks inside. 
+We have listed the checks and their details here for more detailed information.
```
Comment (ademidoff) [inline docs/get-started/advisors-details.md:47]:
```suggestion
We have listed the checks and their details here.
```

**Review by ademidoff — CHANGES_REQUESTED** [review]

File: docs/get-started/advisors-details.md [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+| CVE Security| mongodb\_cve\_version | Shows an error if MongoDB or Percona Server for MongoDB version is older than the latest version containing CVE (Common Vulnerabilities and Exposures) fixes. | MongoDB CVE Version |
+
+
+### MySQL
+| Advisor| Check Name | Description | Summary |
+| :--------- | :---------- | :--- |:--- |
+|Connection Configuration| mysql\_configuration\_max\_connections\_usage | Checks for MySQL max_connections configuration option for maximum utilization | Check max connections usage |
+| Generic Configuration | mysql\_automatic\_sp\_privileges\_enabled | This check reviews the automatic\_sp\_privileges configuration is ON. | Checks if automatic\_sp\_privileges configuration is ON. |
+| Generic Configuration | mysql\_config\_binlog\_retention\_period | Binlogs should not be rotated too often, except very specific cases. | Binlogs retention check |
+| Generic Configuration | mysql\_config\_binlog\_row\_image | Please consider setting binlog\_row\_image=FULL. | Binlogs raw image is not set to FULL |
```
Comment (catalinaadam) [inline docs/get-started/advisors-details.md:96]: The Summary does not match the format of the other ones, which seems to be a Title format. Instead it kind of duplicates the description. Should it be "automatic\_sp\_privileges check"?

Comment (catalinaadam) [inline docs/get-started/advisors-details.md:96]: Checks whether binlogs are being rotated too frequently, which is not recommended except in very specific cases.
```suggestion
| Generic Configuration | mysql\_config\_binlog\_retention\_period | Binlogs should not be rotated too often, except very specific cases.| Binlogs Retention Check |
```

File: docs/get-started/advisors-details.md [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+| InnoDB Configuration| mysql\_configuration\_innodb\_file\_format | Check if InnoDB is configured with recommended file format | MySQL InnoDB file format |
+| InnoDB Configuration| mysql\_configuration\_innodb\_file\_maxlimit | Check if InnoDB is configured with recommended auto-extend | InnoDB Tablespace size has a maximum limit. |
+| InnoDB Configuration| mysql\_configuration\_innodb\_file\_per\_table\_not_enabled | innodb\_file\_per_table not enabled | innodb\_file\_per_table not enabled |
+| InnoDB Configuration| mysql\_configuration\_innodb\_flush\_method | Check if InnoDB is configured with recommended flush method | MySQL InnoDB flush method |
+| InnoDB Configuration| mysql\_configuration\_innodb\_strict\_mode | This check warns about password lifetime. | InnoDB strict mode |
+| Replication Configuration| mysql\_config\_relay\_log\_purge | Identify if a replica node has relay-logs purge set. | Automatic relay log purging is off |
```
Comment (catalinaadam) [inline docs/get-started/advisors-details.md:113]: Is this description correct for this entry?

**Review by BupycHuk — APPROVED** [review]

**Review by catalinaadam — APPROVED** [review]

**Review by ademidoff — APPROVED** [review]

**Pull request merged**